{
    "Computer":  "CRMSERVER2019",
    "Events":  [
                   {
                       "TimeCreated":  "\/Date(1758556582217)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to start instance \"{b4094d3f-0b4d-4597-a951-a42e9e85bf02}\" of \"\\Microsoft\\Windows\\.NET Framework\\.NET Framework NGEN v4.0.30319\"  task for user \"NT AUTHORITY\\SYSTEM\" . Additional Data: Error Value: 2147946720.",
                       "Id":  103,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  239484
                   },
                   {
                       "TimeCreated":  "\/Date(1758556582217)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to complete task \"\\Microsoft\\Windows\\.NET Framework\\.NET Framework NGEN v4.0.30319\" , instance \"{b4094d3f-0b4d-4597-a951-a42e9e85bf02}\" , action \"\" . Additional Data: Error Value: 2147946720.",
                       "Id":  202,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  239483
                   },
                   {
                       "TimeCreated":  "\/Date(1758554735232)\/",
                       "Source":  "Service Control Manager",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "A timeout was reached (30000 milliseconds) while waiting for the CRM Web Interface service to connect.",
                       "Id":  7009,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  453594
                   },
                   {
                       "TimeCreated":  "\/Date(1758554652795)\/",
                       "Source":  "Microsoft-Windows-Kerberos-Key-Distribution-Center",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "This event indicates an attempt was made to use smartcard logon, but the KDC is unable to use the PKINIT protocol because it is missing a suitable certificate.",
                       "Id":  19,
                       "ErrorCode":  null,
                       "UpdateTitle":  null,
                       "EventRecordID":  453404
                   },
                   {
                       "TimeCreated":  "\/Date(1758554581701)\/",
                       "Source":  "Microsoft-Windows-DHCP-Server",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The DHCP server was unable to create or lookup the DHCP Administrators local group on this computer.  The error code is in the data.",
                       "Id":  1036,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  453305
                   },
                   {
                       "TimeCreated":  "\/Date(1758554553865)\/",
                       "Source":  "Microsoft-Windows-DHCP-Server",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The DHCP service was unable to create or lookup the DHCP Users local group on this computer.  The error code is in the data.",
                       "Id":  1035,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  453300
                   },
                   {
                       "TimeCreated":  "\/Date(1758554544041)\/",
                       "Source":  "Service Control Manager",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The SplashtopRemoteService service failed to start due to the following error: \r",
                       "Id":  7000,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  453558
                   },
                   {
                       "TimeCreated":  "\/Date(1758554515064)\/",
                       "Source":  "VSS",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Volume Shadow Copy Service error: Unexpected error calling routine RegOpenKeyExW(-2147483646,SYSTEM\\CurrentControlSet\\Services\\VSS\\Diag,...).  hr = 0x80070005, Access is denied.\r",
                       "Id":  8193,
                       "ErrorCode":  "0x80070005",
                       "UpdateTitle":  null,
                       "EventRecordID":  45221
                   },
                   {
                       "TimeCreated":  "\/Date(1758554497405)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to start \"\\Microsoft\\Windows\\Workplace Join\\Automatic-Device-Join\" task for user \"(default user)\". Additional Data: Error Value: 2147943568.",
                       "Id":  101,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  239278
                   },
                   {
                       "TimeCreated":  "\/Date(1758554461808)\/",
                       "Source":  "Microsoft-Windows-Hyper-V-VmSwitch",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "Failed to restore configuration for port 93E94012-713D-4323-B618-59ACA8D9606C (Friendly Name: ) on switch FFA4CC58-1AD2-455B-967D-44D1669D978E (Friendly Name: ), status = Object Name not found..",
                       "Id":  15,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  453423
                   }
               ],
    "LastPoll":  "09/22 13:21"
}